File: app/src/main/java/com/example/moviemvp/Model/Movie.java
package com.example.moviemvp.Model;

import com.google.gson.annotations.SerializedName;

public class Movie {

    private int id;

    private String title;

    @SerializedName("release_date")
    private String releaseDate;

    @SerializedName("vote_average")
    private float rating;

    @SerializedName("poster_path")
    private String thumbPath;

    @SerializedName("overview")
    private String overview;

    @SerializedName("backdrop_path")
    private String backdropPath;

    @SerializedName("runtime")
    private String runTime;

    @SerializedName("tagline")
    private String tagline;

    @SerializedName("homepage")
    private String homepage;

    public Movie(int id, String title, String releaseDate, float rating, String thumbPath, String overview, String backdropPath, String runTime, String tagline, String homepage) {
        this.id = id;
        this.title = title;
        this.releaseDate = releaseDate;
        this.rating = rating;
        this.thumbPath = thumbPath;
        this.overview = overview;
        this.backdropPath = backdropPath;
        this.runTime = runTime;
        this.tagline = tagline;
        this.homepage = homepage;
    }

    public int getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getReleaseDate() {
        return releaseDate;
    }

    public float getRating() {
        return rating;
    }

    public String getThumbPath() {
        return thumbPath;
    }

    public String getOverview() {
        return overview;
    }

    public String getBackdropPath() {
        return backdropPath;
    }

    public String getRunTime() {
        return runTime;
    }

    public String getHomepage() {
        return homepage;
    }

    @Override
    public String toString() {
        return "Movie{" +
                "id=" + id +
                ", title='" + title + '\'' +
                ", releaseDate='" + releaseDate + '\'' +
                ", rating=" + rating +
                ", thumbPath='" + thumbPath + '\'' +
                ", overview='" + overview + '\'' +
                ", backdropPath='" + backdropPath + '\'' +
                ", runTime='" + runTime + '\'' +
                ", tagline='" + tagline + '\'' +
                ", homepage='" + homepage + '\'' +
                '}';
    }
}

File: app/src/main/java/com/example/moviemvp/model/Movie.kt
package com.example.moviemvp.model

import com.google.gson.annotations.SerializedName

data class Movie(val id: Int, val title: String, @field:SerializedName("release_date")
                 val releaseDate: String, @field:SerializedName("vote_average")
                 val rating: Float, @field:SerializedName("poster_path")
                 val thumbPath: String, @field:SerializedName("overview")
                 val overview: String, @field:SerializedName("backdrop_path")
                 val backdropPath: String, @field:SerializedName("runtime")
                 val runTime: String, @field:SerializedName("tagline")
                 private val tagline: String, @field:SerializedName("homepage")
                 val homepage: String) {

    override fun toString(): String {
        return "Movie{" +
                "id=" + id +
                ", title='" + title + '\''.toString() +
                ", releaseDate='" + releaseDate + '\''.toString() +
                ", rating=" + rating +
                ", thumbPath='" + thumbPath + '\''.toString() +
                ", overview='" + overview + '\''.toString() +
                ", backdropPath='" + backdropPath + '\''.toString() +
                ", runTime='" + runTime + '\''.toString() +
                ", tagline='" + tagline + '\''.toString() +
                ", homepage='" + homepage + '\''.toString() +
                '}'.toString()
    }
}